events/_event-post-macros.html

```diff
@@ -131,19 +168,24 @@
 
    Create HTML event meta date/time markup when given:
 
-   date: A post date from a query result.
+   date:                        A post date from a query result.
+
+   datetime_format (optional):  An object to use for customizing event-meta time and
+                                date formats.
 
    ========================================================================== #}
 
-{% macro event_meta_datetime(date) %}
+{% macro event_meta_datetime(date, datetime_format={}) %}
+    {%- set date_format =  datetime_format.date_format | default('%b %m, %Y', true)   -%}
+    {%- set time_format =  datetime_format.time_format | default('%H:%M %p %Z', true) -%} 
+
     <time class="event-meta_datetime dt-start" datetime="{{ date | date('%Y-%m-%d %H:%M %p %Z') }}">
```

Comment:
So I think the correct pattern is : %Y-%m-%dT%H:%M:%S%Z
Although we will probably be switching to schema so ...

http://microformats.org/wiki/value-class-pattern